src/Core/src/Platform/Windows/SliderExtensions.cs

```diff
@@ -99,21 +103,50 @@ public static void UpdateThumbColor(this Slider platformSlider, ISlider slider)
 			"SliderThumbBackgroundDisabled",
 		};
 
-		internal static async Task UpdateThumbImageSourceAsync(this MauiSlider nativeSlider, ISlider slider, IImageSourceServiceProvider? provider)
+		internal static async Task UpdateThumbImageSourceAsync(this MauiSlider nativeSlider, ISlider slider, IImageSourceServiceProvider? provider, Size? defaultThumbSize)
 		{
 			var thumbImageSource = slider.ThumbImageSource;
 
 			if (thumbImageSource == null)
 			{
 				nativeSlider.ThumbImageSource = null;
+
+				if (defaultThumbSize.HasValue && nativeSlider.GetFirstDescendant<Thumb>() is Thumb thumb)
```

Comment:
Same question, If we need to use is, we might want to look at the GetFirstDescendant implementation

Reply:
Same as before.